common/src/main/java/com/wynntils/features/user/redirects/ChatRedirectFeature.java

```diff
@@ -468,6 +472,25 @@ protected String getNotification(Matcher matcher) {
         }
     }
 
+    private class TeleportationFailRedirector extends SimpleRedirector {
+        private static final Pattern SYSTEM_PATTERN = Pattern.compile("§cThere are aggressive mobs nearby...$");
```

Comment:
This seems like an incomplete regex to me? You have no `^` in the beginning but `$` at the end. This sure works but why use the only one?

Reply:
(and you didn't use any in https://github.com/Wynntils/Artemis/pull/800/)